src/content/blog/astro-5130.mdx

```diff
@@ -0,0 +1,180 @@
+
+Example:
+
+```js
+import { defineConfig } from 'astro/config';
+import sitemap from '@astrojs/sitemap';
+
+export default defineConfig({
+  site: 'https://example.com',
+  integrations: [
+    sitemap({
+      customSitemaps: [
+       'https://example.com/blog/sitemap.xml',
+       'https://example.com/helpcenter/sitemap.xml'
+      ]
+    })
+  ]
+});
+```
+
+Learn more in the [`@astrojs/sitemap` configuration documentation](https://docs.astro.build/en/guides/integrations-guide/sitemap/#configuration).
+
+
+## Hosted error pages
+
+The adapter `@astrojs/node` now supports for specifying a host to load prerendered error pages
+
+By default, if a user defines a custom error page that is prerendered, Astro will load it from the same host as the one that the request is made to. This change allows users to specify a different host for loading prerendered error pages. This can be useful in scenarios such as where the server is running behind a reverse proxy or when prerendered pages are hosted on a different domain.
+
+To use this feature, set the `experimentalErrorPageHost` adapter option in your Astro configuration to the desired host URL. For example, if your server is running on localhost and served via a proxy, you can ensure the prerendered error pages are fetched via the localhost URL:
+
+```js
+import { defineConfig } from 'astro/config';
+import node from '@astrojs/node';
+export default defineConfig({
+  adapter: node({
+    // If your server is running on localhost and served via a proxy,
+    // set the host like this to ensure prerendered error pages are fetched via the localhost URL
+    experimentalErrorPageHost: 'http://localhost:4321'
+  })
+});
+```
+
+For more information on enabling and using this experimental feature, see the [`@astrojs/node` adapter docs](https://docs.astro.build/en/guides/integrations-guide/node/#experimentalerrorpagehost).
+
+
+## Enum support in Astro DB table
+
+The package `@astrojs/db` adds support for enum support for text columns in Astro DB tables.
+
+```ts
+import { column, defineTable } from 'astro:db';
+
+// Table definition
+const UserTable = defineTable({
+	columns: {
+		id: column.number({ primaryKey: true }),
+		name: column.text(),
+        rank: column.text({ enum: ['user', 'mod', 'admin'] }),
+	},
+});
+
+// Resulting type definition
+type UserTableInferInsert = {
+    id: string;
+    name: string;
+    rank: "user" | "mod" | "admin";
+}
+```
+
+## Local support for Cloudflare KV
+
+The adapter `@astrojs/cloudflare` refactors the internal solution that powers Astro Sessions.
+
+The adapter now uses Cloudflare's local support for [Cloudflare KV](https://developers.cloudflare.com/kv/). This internal change is a drop-in replacement and does not require any change to your projectct code.
+
+However, you can now connect to the remote [Cloudflare KV Namespace](https://developers.cloudflare.com/kv/concepts/kv-namespaces/) if desired and use production data during local development.
```

Comment:
```suggestion
 The newest release of the Cloudflare adapter includes local support for [Cloudflare Workers KV](https://developers.cloudflare.com/kv/) when running `astro dev`. This internal refactoring is a drop-in replacement for powering Astro sessions and does not require any change to your existing project code.

However, you can now choose to connect to the remote [Cloudflare KV Namespace](https://developers.cloudflare.com/kv/concepts/kv-namespaces/) if desired and use production data during local development.
```

Reply:
@sarah11918 Only question if we should add a link to `remote` or `production data`, to this Cloudflare doc page (https://developers.cloudflare.com/workers/development-testing/#remote-bindings). I let you decide.